lib/iris/coord_systems.py

```diff
@@ -645,6 +639,104 @@ def as_cartopy_projection(self):
         return self.as_cartopy_crs()
 
 
+class Geostationary(CoordSystem):
+    """
+    An geostationary satellite image map projection.
```

Comment:
`An geostrationary` -> `A geostationary`

Reply:
Without reading I copied from vertical perspective, which had been copied without reading from another coord system, so I managed to double the error! I feel appropriately ashamed 😖